---
title: "<span style = 'font-size: 100px;'>Group 14</span>"
subtitle: "<span style = 'font-size: 60px;'>Final project 14</span>"
format:
  revealjs:
    theme: simple
    width: 1050
    margin: 0.0005
editor: visual
author:
- "<div style = 'text-align: center; font-size: 20px; color: gray'>Julie Brockhoff, s204519</span>"
- "<div style = 'text-align: center; font-size: 20px; color: gray'>Cecilie Kejlberg Leth, s204564</span>"
- "<div style = 'text-align: center; font-size: 20px; color: gray'>Amalie Drud Nielsen, s204560</span>"
- "<div style = 'text-align: center; font-size: 20px; color: gray'>Chrysillis Hellemann Polhaus, s153664</span>"
- "<div style = 'text-align: center; font-size: 20px; color: gray'>Lise Lotte Eriksen, s154123</span>"
---

## Introduction

::: {style="font-size: 25px; margin-top: 30px;"}
-   Human endogenous retroviruses (HERVs) have been implicated in the pathogenesis of cancers
-   Previously, a targeted knockout (KO) of HERV-K showed reduced tumorigenic attributes, such as proliferation, migration, and invasion of ovarian cancer cells
:::

\

::: {style="font-size: 40px; text-align: center;"}
**The aim of our project was to assess the effect of the HERV-K gene in two ovarian cancer cell lines by identifying differentially expressed genes (DEGs)**
:::

## Materials and methods

::: {style="font-size:70%;"}
::: {style="font-size: 25px; margin-top: 30px;"}
##### Data

-   HERV-K was knocked out in two ovarian cancer cell lines: SKOV3 and OVCAR3
-   Next-geneneration mRNA sequencing was done on baseline and HERV-K gene KO cells

##### Methods

-   DEG identification and plotting using R version 4.3.1

    -   with packages: tidyverse (v2.0.0) and patchwork (v1.1.3)

-   KEGG enrichment using clusterProfiler (v4.10.1), biomaRt (v2.58.2) and enrichplot (v1.22.0)

![](flow_chart.png){fig-align="center" width="90%"}
:::
:::

## Loading data & augment

::: {style="font-size:70%;"}
::: {style="font-size: 25px; margin-top: 30px;"}
##### Data load and clean

-   Merging of 4 .txt-files to 1 .tsv-file

-   Genes found in duplicate was removed

##### Augment

-   Log2 fold change (Log2FC) was calculated based on the gene expression

-   A column was added with information about whether the genes are up-regulated, down-regulated or neutral

-   Based on a critera of +/- 2 Log2FC
:::
:::

## Results - Descriptive statistics

::: {style="font-size: 25px; margin-top: 30px;"}
::: columns
::: {.column width="35%"}
::: {style="font-size:70%;"}
28.040 observations (genes)

|                | OVCAR3 | SKOV3 |
|----------------|--------|-------|
| Up-regulated   | 4212   | 270   |
| Neutral        | 23062  | 26850 |
| Down-regulated | 766    | 920   |

**Common genes:**\
**Upregulated:** 59\
e.g. CRYBB2, ANKRD62 -\> both associated with cancer\

**Downregulated:** 15\
e.g. RBAKDN and CDH2 -\> associated with cell division and adhesion
:::
:::

::: {.column width="65%"}
![](../results/04_gene_counts.png){fig-align="center" width="100%"}
:::
:::
:::

## Results - Heatmap

::: {style="font-size: 25px; margin-top: 30px;"}
::: columns
::: {.column width="50%"}
::: {style="font-size:70%;"}
Expression of 70 randomly selected genes are plotted for comparison of the cell lines. ![](../results/05_heatmap_random.png){fig-align="center" width="100%"}

-   Red indicates that the gene expression is upregulated

-   Blue indicates that the gene expression is downregulated
:::
:::

::: {.column width="50%"}
::: {style="font-size:70%;"}
Expression of up- or down-regulated DEGs are plotted for comparison of the two cell lines. ![](../results/05_heatmap_DEGs.png){fig-align="center" width="100%"}
:::
:::
:::
:::

## Results - Scatterplot for OVCAR3

::: {style="font-size: 25px; margin-top: 30px;"}
::: {style="font-size:70%;"}
Scatter plot of correlation between gene expresseion for KO and baseline genes for OVCAR3\
![](../results/06_scatter_OVCAR3.png){fig-align="center" width="80%"}
:::
:::

## Results - Scatterplot for SKOV3

::: {style="font-size: 25px; margin-top: 30px;"}
::: columns
::: {.column width="50%"}
::: {style="font-size:70%;"}
Scatter plot of correlation between gene expresseion for KO and baseline genes for SKOV3\
![](../results/06_scatter_SKOV3.png){fig-align="center" width="100%"}
:::
:::

::: {.column width="50%"}
::: {style="font-size:70%;"}
Scatter plot of correlation between gene expresseion for KO and baseline of SKOV3\
![](../results/06_scatter_zoom_SKOV3.png){fig-align="center" width="100%"}
:::
:::
:::
:::

## Results - t-test

::: {style="font-size: 25px; margin-top: 30px;"}
::: columns
::: {.column width="30%"}
::: {style="font-size:70%;"}
-   Positive correlation between KO and baseline for each cell line

-   t-test for DEGs for each cell line: not significant (OVCAR3: 0.796 & SKOV3: 0.069)

-   Looking at genes specific for cancer: GDF15

    -   Fisher test: p-value = 2.2\*10\^-16

    -   Log2FC OVCAR3: -4.417

    -   Log2FC SKOV3: -0.33
:::
:::

::: {.column width="70%"}
::: {style="font-size:70%;"}
![](../results/07_scatter_MOCK_KO.png){fig-align="center" width="80%"}
:::
:::
:::
:::

## Results - KEGG

::: {style="font-size: 25px;margin-top: 30px;"}
::: {style="font-size:70%;"}
KEGG plots displaying enriched pathways\

![](../results/08_KEGG_OVCAR3.png){fig-align="center" width="70%"}

Cancer, senescence and PI3K/AKT pathways are down-regulated after HERV-K KO in OVCAR3 (ovarian cancer cells)
:::
:::

## Discussion

::: {style="font-size: 25px;margin-top: 30px;"}
::: {style="font-size:70%;"}
::: {style="font-size: 25px; margin-top: 30px;"}
-   No significant pathways were changed after KO of HERV-K in SKOV3
-   Cancer associated pathways were down-regulated after KO of HERV-K in OVCAR3
-   Ovarian cancer gene GDF15 were down-regulated after KO of HERV-K in OVCAR3
:::

\

::: {style="font-size: 40px; text-align: center;"}
**Collectively this suggest that HERV-K might play a role in development of ovarian cancer in atleast one ovarian cancer cell line. More reseach should be done to identify HERV-K as a new therapeutic target for ovarian cancer treatment**
:::
:::
:::
